ksql-serde/src/main/java/io/confluent/ksql/serde/delimited/KsqlDelimitedDeserializer.java

```diff
@@ -92,11 +93,7 @@ private Object enforceFieldType(Schema fieldSchema, String delimitedField) {
       case FLOAT64:
         return Double.parseDouble(delimitedField);
       case STRING:
-        if (delimitedField.startsWith("'") && delimitedField.endsWith("'")) {
```

Comment:
Why get rid of these checks?

Reply:
No need anymore since the read and write is done with the same apache commons package and we currently support default config.